source/strings.tex

```diff
@@ -4760,8 +4760,8 @@
   constexpr basic_string_view() noexcept;
   constexpr basic_string_view(const basic_string_view&) noexcept = default;
   constexpr basic_string_view& operator=(const basic_string_view&) noexcept = default;
-  constexpr basic_string_view(const charT* str);
-  constexpr basic_string_view(const charT* str, size_type len);
+  constexpr basic_string_view(const value_type* str);
+  constexpr basic_string_view(const value_type* str, size_type len);
```

Comment:
I think this change means a deduction guide would be required to deduce the template args when using these constructors.